content/en/docs/gke/pipelines/upgrade.md

```diff
@@ -1,134 +1,55 @@
-the [GCP Deployment Manager](https://console.cloud.google.com/dm/deployments),
-one for the cluster deployment and one for the storage deployment:
-
-<img src="/docs/images/pipelines-deployment-storage1.png" 
-  alt="Deployment Manager showing the storage deployment entry"
-  class="mt-3 mb-3 border border-info rounded">
-
-The entry with the suffix `-storage` creates one PD for the metadata store and
-one for the artifact store:
-
-<img src="/docs/images/pipelines-deployment-storage2.png" 
-  alt="Deployment Manager showing details of the storage deployment entry"
-  class="mt-3 mb-3 border border-info rounded">
-
-## Reinstalling Kubeflow Pipelines
-
-You can delete a Kubeflow cluster and create a new one, specifying
-your existing storage to retrieve the original data in the new cluster.
+| Installation \ Features                 | In-place Upgrade | Reinstallation on the same cluster | Reinstallation on a different cluster | User Customizations across Upgrades (via [Kustomize](https://kustomize.io/)) |
+|-----------------------------------------|------------------|------------------------------------|---------------------------------------|------------------------------------------------------------------------------------|
+| Standalone                              | ✅                | ⚠️ Data is deleted by default.      |                                       | ✅                                                                                  |
+| [Standalone (managed storage)](https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/env/gcp)            | ✅                | ✅                                  | ✅                                     | ✅                                                                                  |
+| full Kubeflow (>=1.1)                   | ✅                | ✅                                  | Needs documentation                   | ✅                                                                                  |
+| full Kubeflow (<1.1)                    |                  | ✅                                  | ✅                                     |                                                                                    |
+| AI Platform Pipelines                   |                  | ✅                                  |                                       |                                                                                    |
+| AI Platform Pipelines (managed storage) |                  | ✅                                  | ✅                                     |                                                                                    |
 
 Notes:
+* Managed storage means storing all the data in managed storage services on Google Cloud. A Google Cloud Storage (GCS) bucket and a Cloud SQL instance is required. Using managed storages make it easy to manage, back up, and restore Kubeflow Pipelines data.
```

Comment:
Suggested revision(Easy is a good word to avoid, but easier can be a good alternative. In this case the text is indicating that the listed tasks are easier than a default option, instead of "easy". Indicating that something should be "easy" can be a problem because tasks that should be easy can be difficult for some users.):

When you deploy Kubeflow Pipelines with managed storage on Google Cloud, you pipeline's metadata and artifacts are stored in Cloud Storage and Cloud SQL. Using managed storage makes it easier to manage, back up, and restore Kubeflow Pipelines data.

Reply:
Nit: Remove (GCS)

Reply:
Added links to products, thanks @8bitmp3 and used @joeliedtke 's narrative.